internal/sql/repository/pipelineConfig/PipelineStatusTimelineRepository.go

```diff
@@ -147,6 +165,18 @@ func (impl *PipelineStatusTimelineRepositoryImpl) FetchTimelineByWfrIdAndStatuse
 	return timelines, nil
 }
 
+func (impl *PipelineStatusTimelineRepositoryImpl) FetchTimelineByInstalledAppVersionHistoryIdAndStatuses(installedAppVersionHistoryId int, statuses []TimelineStatus) ([]*PipelineStatusTimeline, error) {
```

Comment:
statuses can be replaced by some relevant name, and check its usage, should have >0 len check.

Reply:
No need to check for len>0 since the place where it has been used is already hard coded with the statuses that I want to fetch

Reply:
and also, changed the function name to FetchTimelineByInstalledAppVersionHistoryIdAndPipelineStatuses for relevance.